docs/zh/connector-v2/source/InfluxDB.md

```diff
@@ -0,0 +1,250 @@
+# InfluxDB
+
+> InfluxDB source connector
+
+## Description
+
+通过InfluxDB读取外部数据源数据。
+
+## Support InfluxDB Version
+
+- 1.x/2.x
+
+## Support Those Engines
+
+> Spark<br/>
+> Flink<br/>
+> SeaTunnel Zeta<br/>
+
+## Using Dependency
+
+### For Spark/Flink Engine
+
+> 1. 需要确保连接器Jar包 [influxDB connector jar package](https://mvnrepository.com/artifact/org.apache.seatunnel/connector-influxdb) 被放在目录 `${SEATUNNEL_HOME}/connectors/`.
+
+### For SeaTunnel Zeta Engine
+
+> 1. 需要确保连接器Jar包 [influxDB connector jar package](https://mvnrepository.com/artifact/org.apache.seatunnel/connector-influxdb) 被放在目录 `${SEATUNNEL_HOME}/lib/`.
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [ ] [stream](../../concept/connector-v2-features.md)
+- [x] [exactly-once](../../concept/connector-v2-features.md)
+- [x] [column projection](../../concept/connector-v2-features.md)
+- [x] [parallelism](../../concept/connector-v2-features.md)
+- [ ] [support multiple table reading](../../concept/connector-v2-features.md)
+- [x] [parallelism](../../concept/connector-v2-features.md)
+- [x] [support user-defined split](../../concept/connector-v2-features.md)
+
+## Data Type Mapping
+
+|                                 InfluxDB Data Type                                  | SeaTunnel Data Type |
+|-------------------------------------------------------------------------------------|---------------------|
+| BOOLEAN                                                                             | BOOLEAN             |
+| SMALLINT                                                                            | SHORT               |
+| INT                                                                                 | INTEGER             |
+| SMALLINT UNSIGNED<br/>MEDIUMINT<br/>MEDIUMINT UNSIGNED<br/>INT<br/>INTEGER<br/>YEAR | INT                 |
+| BIGINT                                                                              | LONG                |
+| FLOAT<br/>DOUBLE                                                                    | DOUBLE              |
+| STRING                                                                              | STRING              |
+
+## Options
+
+|        name        |  type  | required | default value |
```

Comment:
![image](https://github.com/apache/seatunnel/assets/76689593/6c7be1d3-b9cc-4ea1-8ed2-308d4a166ce4)
Refer to mysql Documentation